reviews.Rmd

```diff
@@ -0,0 +1,583 @@
+
+:::reviewer
+The fact that they subsequently add a Bayesian sauce to the likelihood ratio (by adding a distribution to the Type 1 error rate when a study is not preregistered, instead of simply making some assumption about a point-value of the inflated Type 1 error rate) does not add anything to their argument.
+So, I believe the authors have basically attempted to re-invent Mayo's severity argument.
+:::
+
+The whole second part of the paper deals with the problem of not being able to reduce the theoretical risk to a point estimate.
+The statistical procedure has an objective Type 1 error rate, for which we can come up with a point estimate if we agree on the conditions, which is a starting point for evaluating the theoretical risk of the scientific (!= statistical) procedure.
+Since, a reader does not have access to the whole scientific procedure, they must judge it with uncertainty.
+We find it important to distentengle the statistical procedure, the scientific procedure, and the outside perspective of the reader.
+
+:::reviewer
+The fact that this is not immediately clear is mainly due to missing discussions of the existing literature.
+Point 1: Preregistration can not objectively minimize uncertainty about the Type 1 error rate The main argument of the authors is that risky predictions create persuasive evidence if they turn out to be correct.
+Preregistrations are ideal for communicating the theoretical risk researchers took when testing a claim, and communicating this removes uncertainty in the reader about the theoretical risk the authors took.
+I think this argument is in the right direction, but it is not fully developed.
+I was somewhat disappointed not to see a discussion of my own paper on a conceptual analysis of the value of preregistration, as I had a similar goal as the authors when I wrote my paper, and my arguments are in the same direction, but dare I say, slightly better developed.
+:::
+
+We find ourselves in a curios position, that perhaps may be summarised as "doing the right thing, for the wrong reasons", i.e., we agree with Laken's conclusions about preregistration but we disagree how to come to these conclusions.
+We take a Bayesian/inductionist view, whereas Lakens takes a frequentist/anti-inductionist view, agreeing on the end result but disagreeing on everything that comes before.
+While we find it important that readers are aware, how anti-inductionist deal with preregistration, this paper was not meant as an exercise to compare Bayesian justification for preregistration with its frequentist counterpart.
+
+:::reviewer
+Maybe everyone thinks this of their own papers, so forgive me if I am too biased, but I will attempt to point out some weaknesses in the current paper, sometimes by citing my own paper.
+I hope the authors see how strongly the ideas are related, and forgive me for citing myself.
+:::
+
+Of course, we see the relevance of Laken's work; we met in Eindhoven to discuss an early version of the paper.
```

Comment:
```suggestion
Of course, we see the relevance of Laken's work; we met in Eindhoven to discuss an early version of the paper.
With respect to how well developed our respective arguments are, we would like to point out that we think that a key part of our argument is the mathematical formulation that enables research to actually compute tradeoffs they face during the research process instead of juggling with vague notions of unquantifiable philosophical principles.
```